src/helics/shared_api_library/backup/helics/helics.h

```diff
@@ -571,6 +571,12 @@ HELICS_EXPORT const char* helicsGetBuildFlags(void);
  */
 HELICS_EXPORT const char* helicsGetCompilerVersion(void);
 
+/**
+ * Get an extended version string in json format, containing version info, build flags and compiler info and information about which core
+ * types are available.
+ */
+HELICS_EXPORT const char* helicsGetExtendedVersionString(void);
```

Comment:
`helicsGetVersion` doesn't have the word `String` in the name, Can we change it to `helicsGetExtendedVersion`?

Also, it's not just version information. Maybe the function should be `helicsGetAdditionalInfo`?

Reply:
how about helicsGetSystemInfo, since it gets more about the system than about HELICS itself?

Reply:
Yes, system information or system report seems good to me.